corehq/apps/app_manager/templates/app_manager/partials/modules/case_search_properties.html

```diff
@@ -55,21 +55,21 @@ <h3>
                     </div>
                     <div class="form-group">
                       <label class="control-label">{% trans "Label Field" %}</label>
-                      <input class="form-control" type="text" data-bind="value: property.itemSet.label"/>
+                      <input class="form-control" type="text" data-bind="value: property.itemSet.label" placeholder="{% trans "name" %}"/>
```

Comment:
FYI there's a `trans_html_attr` tag you can use instead of `trans` that will escape any quotes in the translation.

Reply:
Thinking about this more, I don't think these should actually be translated: https://github.com/dimagi/commcare-hq/pull/28047